ert_gui/ertwidgets/models/analysismodulevariablesmodel.py

```diff
@@ -65,14 +63,5 @@ def getVariableValue(
         cls, facade: LibresFacade, analysis_module_name: str, name: str
     ):
         """@rtype: int or float or bool or str"""
-        logger = logging.getLogger(__name__)
         analysis_module = facade.get_analysis_module(analysis_module_name)
-        variable_type = cls.getVariableType(name)
-        if variable_type == float:
-            return analysis_module.getDouble(name)
-        elif variable_type == bool:
-            return analysis_module.getBool(name)
-        elif variable_type == int:
-            return analysis_module.getInt(name)
-        else:
-            logger.error(f"Unknown variable: {name} of type: {variable_type}")
```

Comment:
The replacement here no longer has an: `else`, should perhaps add that?

Reply:
Asserting variable in getvar-function and raising exception if not valid nor supported type